test/Auth/authModule.js

```diff
@@ -0,0 +1,45 @@
+import test from 'ava';
+import { actionTest, reducerTest } from 'redux-ava';
+
+import {
+  changeUsername,
+  loginRequest,
+  reducer,
+} from '../../src/Auth/authModule';
+
+const username = 'ALi';
+
+// Action
+test('Action: changeUsername', actionTest(
+  changeUsername,
+  { username },
+  {
+    type: 'CHANGE_USERNAME',
+    payload: { username },
+  },
+));
+
+test('Action: loginRequest', actionTest(
+  loginRequest,
+  {},
+  {
+    type: 'LOGIN_REQUEST',
+    payload: {},
+  },
+));
```

Comment:
You need another test to prove that it always returns an empty object as the payload.